root/components/ReleaseGroupsList.js

```diff
@@ -0,0 +1,248 @@
+  groupByType,
+  order,
+  seriesItemNumbers,
+  showRatings,
+  sortable,
+}: ReleaseGroupsListHeaderProps) => (
+  <thead>
+    <tr>
+      {$c.user_exists && checkboxes ? (
+        <th style={{width: '1em'}}>
+          <input type="checkbox" />
+        </th>
+      ) : null}
+      {seriesItemNumbers ? <th style={{width: '1em'}}>{l('#')}</th> : null}
+      <th className="year c">
+        {sortable
+          ? (
+            <SortableTableHeader
+              label={l('Year')}
+              name="year"
+              order={order}
+            />
+          )
+          : l('Year')}
+      </th>
+      <th>
+        {sortable
+          ? (
+            <SortableTableHeader
+              label={l('Title')}
+              name="name"
+              order={order}
+            />
+          )
+          : l('Title')}
+      </th>
+      <th className="artist">{l('Artist')}</th>
+      {groupByType ? null : (
+        <th>
+          {sortable
+            ? (
+              <SortableTableHeader
+                label={l('Type')}
+                name="primary_type"
+                order={order}
+              />
+            )
+            : l('Type')}
+        </th>
+      )}
+      {showRatings ? <th className="rating c">{l('Rating')}</th> : null}
+      <th className="count c">{l('Releases')}</th>
+    </tr>
+  </thead>
+);
+
+const ReleaseGroupsListEntry = ({
+  $c,
+  checkboxes,
+  index,
+  groupByType,
+  releaseGroup,
+  seriesItemNumbers,
+  showRatings,
+}: ReleaseGroupsListEntryProps) => (
+  <tr className={loopParity(index)} key={releaseGroup.id}>
+    {$c.user_exists && checkboxes ? (
+      <td>
+        <input
+          name={checkboxes}
+          type="checkbox"
+          value={releaseGroup.id}
+        />
+      </td>
+    ) : null}
+    {seriesItemNumbers ? (
+      <td style={{width: '1em'}}>
+        {seriesItemNumbers[releaseGroup.id]}
+      </td>
+    ) : null}
+    <td className="c">
+      {releaseGroup.firstReleaseDate
+        ? parseDate(releaseGroup.firstReleaseDate).year
+        : '—'}
+    </td>
+    <td>
+      <EntityLink entity={releaseGroup} />
+    </td>
+    <td>
+      {releaseGroup.artistCredit
+        ? <ArtistCreditLink artistCredit={releaseGroup.artistCredit} />
+        : null}
+    </td>
+    {groupByType ? null : (
+      <td>
+        {releaseGroup.typeName
+          ? releaseGroupType(releaseGroup)
+          : null
+        }
+      </td>
+    )}
+    {showRatings ? (
+      <td className="c">
+        <RatingStars entity={releaseGroup} />
+      </td>
+    ) : null}
+    <td className="c">{releaseGroup.release_count}</td>
+  </tr>
+);
+
+const ReleaseGroupsList = ({
+  $c,
+  checkboxes,
+  groupByType,
+  order,
+  releaseGroups,
+  seriesItemNumbers,
+  showRatings,
+  sortable,
+}: ReleaseGroupsListProps) => {
+  const groupedReleaseGroups = groupBy(releaseGroups, 'typeName');
+  return (
+    groupByType ? (
+      Object.keys(groupedReleaseGroups).map((type) => {
```

Comment:
I'd probably make the grouped listing a separate component from the basic list one (and the grouped one could use the basic list one somehow, as the release-group-list seems mostly repeated between them). But it shouldn't block merging this.

Reply:
Started the split, added a TODO to finish the split once we don't need to embed the component from TT.

Reply:
I would rather create a task ticket and refer to it from component calls in TT files.

Reply:
Done (if that's what you meant, anyway)